planex/cmd/mock.py

```diff
@@ -109,7 +109,6 @@ def main(argv=None):
 
     defaults = [
         "--uniqueext", uuid4().hex,
-        "--disable-plugin", "package_state",
```

Comment:
I add a link to where this documented for future reference: https://fedoraproject.org/wiki/Mock/Plugin/PackageState